src/models.py

```diff
@@ -42,7 +42,16 @@
 
 
 class BasePyTorchUNet(Model):
+    """Base class for PyTorch U-net models.
+
+    Args:
```

Comment:
@taraspiotr todo?

Reply:
@jakubczakon my bad, I've just started documenting models, but in this PR there should be only postprocessing docstrings